### PR TITLE
Refinery Update: Syndicate

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
# Weapons
### Liu Fist Set
A 5 hit combo system similar to yakuza

### Shi Misc Set
A set of 3 Shi weapons of different damage types!

### Los Mariachis Set
A set of Low-grade white weapons with a poise mechanic like Kurokumo. 

### Sweeper Set
A set of 4 weapons that all heal and gib on kills

### Thumb Set
Guns that all have an ammo count, and when you melee you get bullets back
Melee weapons are all sidearms.

## New Weapons
Fixer Hammer 32 damage at 1.4x speed

## Weapon Changes
Jeong is now 40% HP for 3x damage intead of 20% for 5x
BOD is now 25% damage.
Fixer Greatsword is now 38 at 2x speed

# Banners
### Adds in Hana banner
A totality of all low-mid association gear, and hana stuff at a 10% chance. Costs 3 boxes, but takes 9 minutes to get a box (as opposed to 3)
 
### Backstreets Banner
Splits Syndicate Banner into Syndicate and Backstreets Banner. 
Contains Rats, Sweepers, District 23, and axe gang.
It's cheap with a small amount of boxes required, but the gear ain't that good.

## Banner Changes:
- Rats moved to backstreets, Yun moved to Leaflet
- District 23 moved to backstreets

# Fixes and changes:
- Replaces misc sprites
- Fixes Yun weapon
- Fullstop pistol has the right sprite

## Changelog
:cl:
add: Refinery update
/:cl:

### CODE REVIEW

File: code/game/objects/items/ego_weapons/non_abnormality/liu.dm [hunk 1]
```diff
@@ -6,15 +6,14 @@
 
 /obj/item/ego_weapon/city/liu/examine(mob/user)
 	. = ..()
-	. += "This weapon kills insane people."
+	. += "<span class='notice'>This weapon kills insane people."
```
Comment (LanceSmites328) [inline code/game/objects/items/ego_weapons/non_abnormality/liu.dm:9]:
```suggestion
	. += "<span class='notice'>This weapon kills insane people.</span>"
```

File: code/game/objects/items/ego_weapons/non_abnormality/liu.dm [hunk 2]
```diff
@@ -84,3 +83,156 @@
...
+
+/obj/item/ego_weapon/city/liu/fist/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>This weapon has light and heavy attacks. Use in hand to activate a heavy attack. Combos are as follows:"
+	. += "<span class='notice'>LLLLL - 5 Hit fast combo, ending in a knockback attack."
+	. += "<span class='notice'>H 	 - Windup fist attack for 1.5x damage and deals massive stamina damage to humans."
+	. += "<span class='notice'>LH 	 - AOE Fire fist attack. This does not kill insane people."
+	. += "<span class='notice'>LLH 	 - High Damage Combo, last attack has a windup and deals 2x damage."
+	. += "<span class='notice'>LLLH  - Deals good damage. Last hit backsteps you 2 tiles."
+	. += "<span class='notice'>LLLLH - High Damage combo, last hit ends in a 2x damage boost with no windup."
```
Comment (LanceSmites328) [inline code/game/objects/items/ego_weapons/non_abnormality/liu.dm:112]:
```suggestion
	. += "<span class='notice'>This weapon has light and heavy attacks. Use in hand to activate a heavy attack. Combos are as follows:</span>"
	. += "<span class='notice'>LLLLL - 5 Hit fast combo, ending in a knockback attack.</span>"
	. += "<span class='notice'>H 	 - Windup fist attack for 1.5x damage and deals massive stamina damage to humans.</span>"
	. += "<span class='notice'>LH 	 - AOE Fire fist attack. This does not kill insane people.</span>"
	. += "<span class='notice'>LLH 	 - High Damage Combo, last attack has a windup and deals 2x damage.</span>"
	. += "<span class='notice'>LLLH  - Deals good damage. Last hit backsteps you 2 tiles.</span>"
	. += "<span class='notice'>LLLLH - High Damage combo, last hit ends in a 2x damage boost with no windup.</span>"
```

File: code/game/objects/items/ego_weapons/non_abnormality/mariachi.dm [hunk 3]
```diff
@@ -0,0 +1,47 @@
+//Los Mariachis - Grade 7 with poise crits, white version of Kurokumo.
+/obj/item/ego_weapon/city/mariachi
+	name = "maracas (single)"
+	desc = "A single maracas used by Los Mariachis."
```
Comment (LanceSmites328) [inline code/game/objects/items/ego_weapons/non_abnormality/mariachi.dm:4]:
```suggestion
	name = "maraca"
	desc = "A single maraca used by Los Mariachis."
```

File: code/game/objects/items/ego_weapons/non_abnormality/mariachi.dm [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+	special = "This weapon gains 1 poise for every attack. 1 poise gives you a 2% chance to crit at 3x damage, stacking linearly. Critical hits reduce poise to 0."
```
Comment (LanceSmites328) [inline code/game/objects/items/ego_weapons/non_abnormality/mariachi.dm:5]: Note: Not a requested change.
Optional that we make this like Limbus where crits at 20% "true" damage, not benefiting from weaknesses or resistances. In our case it'd likely be Brute.

File: code/game/objects/items/ego_weapons/non_abnormality/mariachi.dm [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+	damtype = WHITE_DAMAGE
+	armortype = WHITE_DAMAGE
+	attack_verb_continuous = list("bashes", "clubs")
+	attack_verb_simple = list("bashes", "clubs")
+	hitsound = 'sound/weapons/fixer/generic/club1.ogg'
+	var/poise = 0
+
+/obj/item/ego_weapon/city/mariachi/examine(mob/user)
+	. = ..()
+	. += "Current Poise: [poise]/20."
```
Comment (LanceSmites328) [inline code/game/objects/items/ego_weapons/non_abnormality/mariachi.dm:17]: Why does it cap?

File: code/game/objects/items/ego_weapons/non_abnormality/mariachi.dm [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+	if(prob(poise*2))
+		force*=3
+		to_chat(user, "<span class='userdanger'>Critical!</span>")
+		poise = 0
+	..()
+	force = initial(force)
+
+//Leader, Grade 6 (She's pretty weak)
+/obj/item/ego_weapon/city/mariachi/dual
+	name = "maracas (double)"
```
Comment (LanceSmites328) [inline code/game/objects/items/ego_weapons/non_abnormality/mariachi.dm:36]:
```suggestion
	name = "maracas"
```

File: code/game/objects/items/ego_weapons/non_abnormality/mariachi.dm [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+	desc = "A pair of maracas used by the leader of Los Mariachis."
+	icon_state = "dualmaracas"
+	force = 19		//Double the maracas twice the attack speed.
+	attack_speed = 0.5
```
Comment (LanceSmites328) [inline code/game/objects/items/ego_weapons/non_abnormality/mariachi.dm:40]: Could probably buff this by giving it a forced follow-up on attack and making it require two hands. After all, holding two maracas in one hand is weird...

File: code/game/objects/items/ego_weapons/non_abnormality/shi.dm [hunk 8]
```diff
@@ -118,3 +118,26 @@
...
+	armortype = WHITE_DAMAGE
+
+/obj/item/ego_weapon/city/shi_assassin/serpent
+	name = "shi association seperant blade"
+	desc = "A unique specialized assassin blade that is used by Shi Section 2. Created for highly armored targets, this one deals black damage"
+	icon_state = "shi_serpent"
+	damtype = BLACK_DAMAGE
+	armortype = BLACK_DAMAGE
+
+/obj/item/ego_weapon/city/shi_assassin/yokai
```
Comment (LanceSmites328) [inline code/game/objects/items/ego_weapons/non_abnormality/shi.dm:137]: This one already does pale, it's special should be buffed accordingly to make up for the 24 lost force, 48 when the ability is used.

File: code/game/objects/items/ego_weapons/non_abnormality/thumb.dm
Comment (LanceSmites328) [inline code/game/objects/items/ego_weapons/non_abnormality/thumb.dm:1]: As is, these weapons kind of miss the point of the Thumb's specialized ammunition, being their Flame, Frost/Cold, and Armor Piercing bullets. While base Ruina uses their melee pages to generate the ammunition that they optionally spend when shooting, we could just have reloading give them a random ammo type.

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/ego_weapons/non_abnormality/liu.dm (hunk 2 above)
Comment (Kitsunemitsu) [reply code/game/objects/items/ego_weapons/non_abnormality/liu.dm:112]: It works as written btw, just a heads up

File: code/game/objects/items/ego_weapons/non_abnormality/mariachi.dm (hunk 5 above)
Comment (Kitsunemitsu) [reply code/game/objects/items/ego_weapons/non_abnormality/mariachi.dm:17]: Kurokumo does too.

File: code/game/objects/items/ego_weapons/non_abnormality/shi.dm (hunk 8 above)
Comment (Kitsunemitsu) [reply code/game/objects/items/ego_weapons/non_abnormality/shi.dm:137]: Fixed

**Comment by InsightfulParasite** [comment]
I have bad news and good news. The changes you made to Tegu Items delete the new sprite states for detector. GOOD NEWS is that i have all 8 of the sprites in a DMI that i can send you on discord.

**Review by LanceSmites328 — APPROVED** [review]

**Pull request merged**